.github/FUNDING.yml

```diff
@@ -0,0 +1 @@
+open_collective: thewpcc/contribute/wp-php-63406
```

Comment:
If we're not sure if this will work, what about using the `custom: LINK1` format ? That should work for sure.

Ref: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository

Reply:
Yeah, I was thinking about that. Should I leave the open collective link to point to the thewpcc, and add the custom link?